src/modules/facility-activation-transaction/dto/create-facility-activation-transaction-request.dto.ts

```diff
@@ -0,0 +1,36 @@
+import { ENUMS } from '@ukef/constants';
+import { ValidatedFacilityIdentifierApiProperty } from '@ukef/decorators/validated-facility-identifier-api-property';
+import { ValidatedNumberApiProperty } from '@ukef/decorators/validated-number-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { UkefId } from '@ukef/helpers';
+
+export type CreateFacilityActivationTransactionRequest = CreateFacilityActivationTransactionRequestItem[];
+
+export class CreateFacilityActivationTransactionRequestItem {
+  @ValidatedFacilityIdentifierApiProperty({
+    description: 'The identifier of the facility to activate.',
+  })
+  readonly facilityIdentifier: UkefId;
+
+  @ValidatedNumberApiProperty({
+    description: 'In most situations value should be 3, it means auto approval.',
+    example: ENUMS.BUNDLE_STATUSES.SUBMIT_FOR_POSTING,
+    minimum: 0,
+    enum: ENUMS.BUNDLE_STATUSES,
+  })
+  readonly initialBundleStatusCode: number;
+
+  @ValidatedStringApiProperty({
+    description: 'In most situations value should be 100, it means first level obligor.',
+    example: ENUMS.LENDER_TYPE_CODES.FIRST_LEVEL_OBLIGOR,
+    length: 3,
+    enum: ENUMS.LENDER_TYPE_CODES,
+  })
+  readonly lenderTypeCode: string;
```

Comment:
Can we use the enum for the type here, instead of `string`?

Reply:
Looks like we didn't use it before, so helpers needs updating. Lets do it as separate PR?

Reply:
makes sense, thanks